src/python/README.md

```diff
@@ -0,0 +1,87 @@
+# Force Submit Proctor Utility
+## Instruction Manual
+
+### Description
+This command line utility calls Proctor's 'expire exams' endpoint repeatedly until there are no more exams to expire. It will then print a csv data table to standard out containing details about all the exams that were expired during the session. All diagnostic output is written to standard error.
+
+#### force_submit.py
+This is the utility script to run. It takes no arguments and is meant to be run from crontab. Because only the csv data goes to standard out, you can run it like this to create a csv file containing all the expired exam details:
+```
+$ ./force_submit.py > recently_expired_exams.csv
+```
+
+#### settings_secret.py
```

Comment:
Can we add something here that explains the URL that should be used to call force submit?  Something like,

The endpoint for force submit in Proctor is currently `http://<PROCTOR URL>/proctor/Services/exams/expire/`.  You will need to replace <PROCTOR URL> to point to whatever Proctor you have deployed.`